config/samples/keystone_v1beta1_keystoneapi.yaml

```diff
@@ -3,6 +3,7 @@ kind: KeystoneAPI
 metadata:
   name: keystone
 spec:
+  containerImage: quay.io/podified-antelope-centos9/openstack-keystone:current-podified
```

Comment:
We shouldn't add explicit `containerImage`s in the samples, as we handle these defaults elsewhere